packages/main/src/Input.js

```diff
@@ -798,6 +801,16 @@ class Input extends UI5Element {
 		};
 	}
 
+	_announceSelectedItem() {
```

Comment:
In this function change the value of the new property. This will invalidate and set it. Don't manipulate DOM directly.